spec/factories/appeal.rb

```diff
@@ -251,7 +251,14 @@
     trait :with_evidence_submission_window_task do
       after(:create) do |appeal, _evaluator|
         root_task = RootTask.find_or_create_by!(appeal: appeal, assigned_to: Bva.singleton)
-        EvidenceSubmissionWindowTask.create!(appeal: appeal, parent: root_task)
+        if appeal.docket_type == Constants.AMA_DOCKETS.hearing
+          hearing_task = appeal.tasks.find_by(type: "HearingTask")
+          parent = hearing_task
+        else
+          parent = root_task
+        end
```

Comment:
I think this is more Ruby-esk:
```suggestion
        parent = root_task
        parent = appeal.tasks.find_by(type: "HearingTask") if appeal.docket_type == Constants.AMA_DOCKETS.hearing
```

Or this may be better:
```suggestion
        parent = appeal.tasks.open.find_by(type: "HearingTask")
        parent ||= root_task
```
I added `open` in case other factory traits add a closed `HearingTask` 🤷 .